features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/admin-dashboard/site/conf/site.json

```diff
@@ -24,7 +24,10 @@
         "identityAlias" : "",
         "responseSigningEnabled":"true",
         "keyStoreName" :""
+    },
+    "customStoreThemeConfig" : {
+        "whitelist" : ["css", "svg"],
+        "themeTypes" : ["default","webapp","site","mobileapp"]
```

Comment:
We can remove this property since the value of the property can be derived from existing configs.